sw/host/tests/manuf/manuf_cp_unlock_raw/src/main.rs

```diff
@@ -0,0 +1,160 @@
+use std::iter;
+use std::rc::Rc;
+use std::time::Duration;
+
+use anyhow::{bail, Context};
+use structopt::StructOpt;
+
+use opentitanlib::chip::boolean::MultiBitBool8;
+use opentitanlib::dif::lc_ctrl::{
+    DifLcCtrlState, DifLcCtrlToken, LcCtrlReg, LcCtrlStatus, LcCtrlTransitionCmd,
+    LcCtrlTransitionCtrl,
+};
+use opentitanlib::io::jtag::{Jtag, JtagParams, JtagTap};
+use opentitanlib::test_utils::init::InitializeTest;
+use opentitanlib::test_utils::poll;
+
+#[derive(Debug, StructOpt)]
+struct Opts {
+    #[structopt(flatten)]
+    init: InitializeTest,
+    #[structopt(flatten)]
+    jtag: JtagParams,
+}
+
+// Verify transition from `RAW` to `TEST_UNLOCKED` `lc_state`.
+//
+// - [x] Pre-load OTP with `RAW` lc_state.
+// - [x] Bring device out of reset selecting the Life Cycle (LC) TAP interface using strap pins.
+//
+// Perfom the following steps via LC TAP interface:
+//
+// - [x] Switch to external clock via `TRANSITION_CTRL.EXT_CLOCK_EN`.
+// - [~] Provide `RAW_UNLOCK`.
+// - [x] Poll for lc_ctrl `STATUS` register for `TRANSITION_SUCCESSFUL`. Check for errors or timeout.
+// - [~] Perform reset by toggling the `POR` pin.
+// - [x] Verify the `lc_state` is `TEST_UNLOCKED0`.
+//
+// The manufacturing environment is required to support test reruns whenever possible:
+//
+// - [ ] Verify that the `RAW_UNLOCK` process can be re-tried after failed or interrupted attempts.
+
+fn main() -> anyhow::Result<()> {
+    let opts = Opts::from_args();
+    opts.init.init_target()?;
+
+    let transport = opts
+        .init
+        .init_target()
+        .context("failed to initialise target")?;
```

Comment:
Same as [here](https://github.com/lowRISC/opentitan/pull/17863#discussion_r1171716307), I think some of these are a bit noisy and provide little value over what the backtrace will provide. Some (like the ones used for functions called at several places in the test, i.e., `reset_target`) may be useful.

Reply:
Removed the noisiest ones and happy to remove the rest if necessary